### PR TITLE
Enable baseline-qemu-docker test plan on lab-collabora-staging

### PR DESCRIPTION
**Pull request by mgalka**
This one is to enable temporarily baseline-qemu-docker on lab-collabora-staging

### CODE REVIEW

File: config/core/lab-configs.yaml [hunk 1]
```diff
@@ -55,7 +55,8 @@ labs:
     lab_type: lava
     url: 'https://staging.lava.collabora.dev/RPC2/'
     priority: '45'
-    filters: *collabora-filters
+    filters:
+      tree: [android]
```
[the remark this answers is not in the document]
Comment (mgalka) [reply config/core/lab-configs.yaml]: yes, corrected

**Comment by gctucker** [comment]
@mgalka Please update the commit subject with `lab-configs.yaml: ` prefix and add a description of what this does.  Is the staging lab using a recent enough LAVA version but not production (2021.05)?  Then we should be able to merge this.

**Comment by gctucker** [comment]
Also, for some reason it doesn't seem to be working as no results appear on the dashboard:
https://staging.kernelci.org/test/job/kernelci/branch/staging-mainline/kernel/staging-mainline-20211217.0/plan/baseline-qemu-docker/

**Comment by gctucker** [comment]
Ah the `qemu` device for `lab-collabora-staging` was offline.  It has worked in the past:
https://staging.kernelci.org/test/job/kernelci/branch/staging-stable/kernel/staging-stable-20211215.0/plan/baseline-qemu-docker/

**Comment by gctucker** [comment]
@mgalka Looks like this is working as expected now:
https://staging.lava.collabora.dev/scheduler/device_type/qemu?dt_length=25&dt_search=docker#dt_

**Comment by gctucker** [comment]
Results are looking fine on the dashboard too:
https://staging.kernelci.org/test/job/kernelci/branch/staging-mainline/kernel/staging-mainline-20220112.0/plan/baseline-qemu-docker/

**Review by gctucker — APPROVED** [review]
LGTM, but please fix the commit message

**Comment by gctucker** [comment]
Never mind, this has been superseded by #1015 now.

**Pull request closed without merging**